data-es6.js

```diff
@@ -4979,7 +4983,7 @@ exports.tests = [
         return result[l-3] === sym1 && result[l-2] === sym2 && result[l-1] === sym3;
       */},
       res: {
-        babel:       { val: false, note_id: "forin-order", note_html: "This uses native for-in enumeration order, rather than the correct order." },
```

Comment:
Won't removing this note_html without adding it back in elsewhere remove the note for other `forin-order` notes?